flexbox/build.gradle

```diff
@@ -37,6 +37,10 @@ android {
             minifyEnabled false
         }
     }
+
+    libraryVariants.all {
+        it.generateBuildConfig.enabled = false
```

Comment:
It would be better to use the following:
```suggestion
        it.getGenerateBuildConfigProvider().configure {
            it.enabled = false
        }
```

Reply:
When I created this PR `getGenerateBuildConfigProvider` was not available. I will rebase and switch to the new method because AGP has since been updated to 3.5.0.